Resources/views/standard_layout.html.twig

```diff
@@ -281,7 +282,7 @@ file that was distributed with this source code.
                                                 {% endif %}
 
                                                 {% block sonata_admin_content_actions_wrappers %}
-                                                    {% if _actions|replace({ '<li>': '', '</li>': '' })|trim is not empty %}
+                                                    {% if _actions|replace({ '<li>': '', '</li>': '' }) is not empty %}
```

Comment:
But given this _actions:

```
     <li>  </li>     
```

first trim will leave it like that:

```
<li>  </li>
```

But this second trim will leave it like that: '' (empty)